sysutils/btop/Portfile

```diff
@@ -3,46 +3,27 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           makefile 1.0
-PortGroup compiler_blacklist_versions 1.0
+PortGroup           compiler_blacklist_versions 1.0
 
-github.setup        aristocratos btop 7e50b03e391a346b2b6ce92be332c58d91f75a00
-version             20230615
-
-fetch.type          git
+github.setup        aristocratos btop 1.3.0 v
```

Comment:
this version is likely not considered "newer" by MacPorts - please verify (i.e., is the port considered "outdated" if you have the previous version installed?). I doubt that is the case in which you should add "epoch 1".

Reply:
Can confirm that MacPorts does not consider this version newer without `epoch 1`, which does fix the problem